src/app/debug/filter-side-drawer/filter-side-drawer.component.html

```diff
@@ -0,0 +1,55 @@
+@if (shouldShowFilter) {
+  <div @removeTrigger class="filter-vertical-panel" data-cy-debug="filter-side-drawer">
+    <div class="panel-container">
+      <h1 class="h1 filter-label">Filter</h1>
+      <div class="filter-input-container">
+        @for (metadataName of metadataNames; track metadataNames) {
```

Comment:
The track expression should be used to identify each element instead of the collection to increase performance, check [the docs](https://angular.io/api/core/for#description) for more info
```suggestion
        @for (metadataName of metadataNames; track metadataName) {
```